modules/account-lib/src/coin/algo/transferBuilder.ts

```diff
@@ -8,7 +8,8 @@ import { TransactionBuilder } from './transactionBuilder';
 import { Transaction } from './transaction';
 import { TransferTransactionSchema } from './txnSchema';
 import Utils from './utils';
-
+import crypto from 'crypto';
+import _ from 'lodash';
```

Comment:
nit: add extra line between imports and class

Reply:
done! message not deprecadet becasue witheline is next to it